gen-all-repos.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/python3
+# pylint: disable=invalid-name
+"""
+Generate all repo snapshot configuration files based on the definitions list
+"""
+
+import argparse
+import json
+import logging
+import subprocess
+import sys
+
+
+logging.basicConfig(level=logging.INFO, format='%(message)s')
```

Comment:
This is going to *highly* unlikely but please move this call into `main()` so you don't configure loggers if someone imports this.